.travis.yml

```diff
@@ -2,51 +2,25 @@ language: python
 sudo: false
 services:
 - docker
+- xvfb
 python:
 - '3.6'
-addons:
-  postgresql: '10'
-  apt:
-    update: true
-    packages:
-    - postgresql-10
-    - postgresql-client-10
-    - default-jre # Java11 is used for elasticmq
-    - libgnutls28-dev
-env:
-  global:
-  - PGPORT=5432
 cache:
   directories:
   - "$HOME/.cache/pip"
-before_cache:
-- rm -f $HOME/.cache/pip/log/debug.log
-before_script:
-- npm install
-- npm install -g bower
-- npm install -g gulp
-- npm install -g karma-cli
-- bower install
-- gulp dev
-- sudo sed -i 's/port = 5433/port = 5432/' /etc/postgresql/10/main/postgresql.conf
-- sudo cp /etc/postgresql/{9.4,10}/main/pg_hba.conf
-- sudo service postgresql restart
-- psql -c "CREATE DATABASE evalai" -U postgres
-- wget https://s3-eu-west-1.amazonaws.com/softwaremill-public/elasticmq-server-0.14.2.jar
-- java -jar elasticmq-server-0.14.2.jar &
 before_install:
 - sudo rm -f /etc/boto.cfg
 - export CHROME_BIN=chromium-browser
 - export DISPLAY=:99.0
-services:
-- xvfb
 install:
-- pip install -r requirements/dev.txt
-- pip install awscli==1.16.57 coveralls
+- pip install awscli==1.18.66 flake8==3.8.2
 script:
 - flake8 ./
-- karma start --single-run
-- py.test --cov . --cov-config .coveragerc
+- docker-compose build
+- docker-compose run nodejs karma start --single-run
```

Comment:
@Sanji515 can I get your help here please regarding running the frontend tests inside docker container. It seems to work fine on my local machine but seems to fail on Travis CI with the following error:

```
6.27s$ docker-compose run nodejs karma start --single-run
Creating network "evalai_default" with the default driver
23 05 2020 09:38:02.076:WARN [filelist]: Pattern "/code/frontend/dist/vendors/*.js" does not match any file.
23 05 2020 09:38:02.093:WARN [filelist]: Pattern "/code/frontend/dist/js/config.js" does not match any file.
23 05 2020 09:38:02.380:INFO [karma-server]: Karma v4.4.1 server started at http://0.0.0.0:9876/
23 05 2020 09:38:02.381:INFO [launcher]: Launching browsers ChromeWithNoSandbox with concurrency unlimited
23 05 2020 09:38:02.387:INFO [launcher]: Starting browser ChromeHeadless
23 05 2020 09:38:02.610:INFO [HeadlessChrome 78.0.3882 (Linux 0.0.0)]: Connected on socket 514jYhIr6RZBeS1DAAAA with id 78782080
HeadlessChrome 78.0.3882 (Linux 0.0.0): Executed 0 of 0 SUCCESS (0 secs / 0 secs)
HeadlessChrome 78.0.3882 (Linux 0.0.0) ERROR
  An error was thrown in afterAll
  ReferenceError: angular is not defined
      at <Jasmine>
      at Suite.<anonymous> (frontend/tests/controllers-test/challengeHostTeamsCtrl.test.js:4:16)
      at <Jasmine>
      at frontend/tests/controllers-test/challengeHostTeamsCtrl.test.js:3:1
HeadlessChrome 78.0.3882 (Linux 0.0.0): Executed 0 of 0 ERROR (0 secs / 0 secs)
e 78.0.3882 (Linux 0.0.0): Executed 0 of 0 ERROR (0.019 secs / 0 secs)
```

Build link: https://travis-ci.org/github/Cloud-CV/EvalAI/builds/690305470#L160

Would appreciate your help on this. Thanks!

Reply:
Hey @deshraj I think it may not be able to get the correct path of the angular files as you are running the tests inside docker so you might also have to modify the path of the files inside karma configuration as inside docker container all the files are inside `/code` directory.

Reply:
But as it was running locally inside docker container with these configurations only so might be some other problem but it is worth checking if it's getting the right path of the files.

Reply:
Ah ok. That is possible. Will definitely try that. Thanks